Automattic/wp-calypso
# Add/sensei launchpad

#### Proposed Changes

* Implements basic Sensei Launchpad

#### Testing Instructions



* Checkout this branch and start the calypso locally running `yarn start`
* Visit `http://calypso.localhost:3000/setup/sensei/` on your browser. This should kickstart the Sensei Tailored Flow on Dotcom.
* Go through the steps. You can select a free domain and a monthly plan.
* You will land on the checkout screen. Make the payment.
* Confirm you land on the Sensei Launchpad screen.
* Copy the browser address url when you're on the Sensei Launchpad screen. You'll need it later for convenience. 
* Confirm all the links work correctly
  * The Domain "Customize" badge/button lands user on the domain selection screen.
  * The "Choose a Plan" link/badge/button lands user on the plan selection screen.
  * "Publish your first Course" link lands user in new course post editor page. **Note:** This works only after the second attempt. I still can't figure out how to land the user to the custom admin page for the very first visit. See p6rkRX-4VM-p2#comment-5127 and pdNBB9-am-p2#screen-final. Publish a course.
* Confirm the "Publish your first Course" task status changes to complete when user publishes their first course. This needs some changes from the Sensei LMS site. Follow [these instructions](https://github.com/Automattic/sensei/pull/6124) on how to test this.

#### Pre-merge Checklist





Related to [Automattic/sensei-pro#1871](https://github.com/Automattic/sensei-pro/issues/1871)
